db/migrate/20210723095821_devise_create_users.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :nickname,           null: false
+      t.string :email,              unique: false, default: ""
```

Comment:
```ruby
t.string :email, unique: false, default: ""
```
申し訳ございません。
README.mdにおいて、指摘もれがございました。
emailカラムにオプションで`unique: false`を付与してしまうと、同じメールアドレスでの登録が可能になってしまいます。
`unique: true`で実装いただければ幸いです。

また、45行目にunique: trueを実装するようデフォルトで設定されているため、
マイグレーションファイルへ記述する必要はありません。

README.mdも併せて修正していただけると幸いです。
お手数をおかけいたしますが、よろしくお願いいたします。

Reply:
migrationファイルは変えずにREADMEファイルだけ変えればよろしいですか？

Reply:
> migrationファイルは変えずにREADMEファイルだけ変えればよろしいですか？

指摘が分かりづらく、申し訳ございません。
README.mdを変更していただき、ありがとうございます！

```suggestion
      t.string :email,              default: ""
```
migaratinファイルに関しては、上記のように`unique: false`の削除していただきたく存じます。
お手数をおかけいたしますが、よろしくお願いいたします。